test/asm/math.asm

```diff
@@ -0,0 +1,24 @@
+	assert 2*10**2*2 == 400
```

Comment:
Thanks for the tests, and they're overall good; however, they only tests the "constexpr" version of the computations; a second set of tests (should be easy to make using a macro) should ensure copies of the computations are deferred to RGBLINK, and tested. (No need for `.out.bin` for that, `assert` should do the trick.)

As for how to force deferring, add `X` to all function arguments, define `X` as a label *after* everything, and don't fix its `SECTION`'s address. That should be bulletproof enough.
To process arguments more easily, I'd suggest the following macro:
```
test: MACRO
ARGS equs "\1(\2) + X"
REPT _NARG - 2
    shift
TMP equs "{ARGS}, (\2) + X"
    purge ARGS
ARGS equs "{TMP}"
    purge TMP
ENDR

; Test RGBASM
X equ 0
    static_assert ARGS \2
    purge X
; Test RGBLINK
    assert ARGS \2
ENDM

    test DIV(, 5.0, 2.0, ) == 2.5
```

Reply:
Done.